src/main/scala/nl.knaw.dans.easy.dd2d/DepositIngestTask.scala

```diff
@@ -71,6 +82,15 @@ case class DepositIngestTask(deposit: Deposit,
     // TODO: delete draft if something went wrong
   }
 
+  def moveDepositToOutbox(subDir: OutboxSubdir): Unit = {
+    try {
+      deposit.dir.copyToDirectory(File(outboxDir.toString.concat(subDir.toString)))
+      deposit.dir.delete()
+    } catch {
+      case e: Exception => logger.info(s"Failed to move deposit: $deposit to the designated outbox : $e")
+    }
```

Comment:
I opted for catching the exception and loggig the error because I thought that the import  shouldn't fail only because the deposit could not be moved to the outbox.

Reply:
@ChrisMuller80 : I think we discussed this. This actually a rather essential point. If we are going to process terabytes of data files this way it will be a huge waste of processing and storage resources.

Reply:
I had started the refactoring. I should have put the PR state back to draft.